weaver/cli.py

```diff
@@ -939,6 +939,8 @@ def _update_files(self, inputs, url=None):
                         "Cannot upload to vault: [%s]", file
                     )
                     continue
+                if not href.startswith("/"):  # Check to prevent upload of external references (http, https, etc.)
+                    continue
```

Comment:
I wonder if I provide `-I input:File=relative/path/file.txt` to the CLI, will this change cause the relative file that would normally be found to fail? Would separating the previous `if "://" not in href and not os.path.isfile(href):` produce the intended use case (must be a local file that exists)?

Reply:
Interesting. I'll try it out.

I tried separating the previous `if` while removing this one, and it caused my unit tests to fail (those that feed an input with NamedTemporaryFiles paths).

In the meantime, I tried this and so far it seems to work with:

```
if "://" not in href and not os.path.isfile(href):
                    LOGGER.warning(
                        "Ignoring potential local file reference since it does not exist. "
                        "Cannot upload to vault: [%s]", file
                    )
                    continue
                if not os.path.isfile(href):  # Check to prevent upload of external references (http, https, etc.)
                    continue
```

Reply:
Replace the `and` by `or` ?  Since `file://` is removed just before, if either `://` is still there or the path does not resolve, then it is not a valid local file.

Reply:
Tested the new change we talked about and everything seemed fine with the other tests. Updated the `LOGGER.warning` to reflect the change, but I was wondering if it might not be better to change it to `info`, since it is also now an expected outcome?

Reply:
I prefer to have it as warning since, although the outcome is expected, the problematic link was most probably added by mistake by the user. This would quickly highlight the problem and what must be fixed if the process fails afterward.